frontends/mit-open/src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -329,6 +333,28 @@ const UserListDetailsTab: React.FC<UserListDetailsTabProps> = (props) => {
   )
 }
 
+const TopPicksCarousel: React.FC = () => {
+  const { isLoading: isLoadingProfile, data: profile } = useProfileMeQuery()
+  const config = TopPicksCarouselConfig(profile)
+  const { data, isLoading } = useLearningResourcesSearch(
+    config[0].data
+      .params as LearningResourcesSearchApiLearningResourcesSearchRetrieveRequest,
```

Comment:
Do we have a shorter equivalent?

Reply:
No 😭

I import `LearningResourcesSearchApiLearningResourcesSearchRetrieveRequest as SearchRequest` sometimes.